aldryn_faq/boilerplates/bootstrap3/templates/aldryn_faq/plugins/question_list.html

```diff
@@ -1,3 +1,5 @@
+{% load i18n %}
```

Comment:
i think we should not load libs that are not used in this particular template.
although it's fairly minor, it still is performance hit